apps/frontend/src/app/3_organisms/StakeForm/components/AddStakeForm/AddStakeForm.tsx

```diff
@@ -232,7 +236,7 @@ export const AddStakeForm: FC<StakeFormProps> = ({
           />
         )}
         <SimpleTableRow
-          label={t(translations.stakePage.stakeForm.votingPowerReceived)}
+          label={t(translations.stakePage.stakeForm.votingPowerIncrease)}
```

Comment:
Delete `stakeForm.votingPowerReceived` from `translations` as it's not used anywhere.